app/serializers/api/admin/order_serializer.rb

```diff
@@ -28,15 +28,9 @@ def payments_path
     spree_routes_helper.admin_order_payments_path(object)
   end
 
-  def ship_path
-    spree_routes_helper.fire_admin_order_path(object, e: 'ship')
-  end
-
-  def payment_capture_path
+  def ready_to_capture
     pending_payment = object.pending_payments.first
-    return '' unless object.payment_required? && pending_payment
-
-    spree_routes_helper.fire_admin_order_payment_path(object, pending_payment.id, e: 'capture')
+    object.payment_required? && pending_payment
```

Comment:
this code has no unit tests so far.

Reply:
There's a quick check that the value is included in the serialised object. Do you think this line deserves it's own test?

Reply:
I couldnt find api/admin/order_serializer_spec.rb

Reply:
Ah, the output of the serializer is tested in: `spec/controllers/api/orders_controller_spec.rb`.

Reply:
ah, ok.
I think we should have unit tests for the serializers. I dont think it's a requirement for this PR, I just thought of mentioning it.